pkg/descheduler/descheduler.go

```diff
@@ -19,7 +19,7 @@ package descheduler
 import (
 	"fmt"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
why this change?

Reply:
I believe VS Code made this change automatically. I can change it back.

Reply:
Fixed.